### DISCUSSION TITLE
after upgrading to nextjs 12 facing issue

### DISCUSSION THREAD

**Discussion by vikas990** [react-hook-form/react-hook-form#7419] [Q&A] [answered]
hi, I have updated my next js version to 12 and after upgrading, I m facing the issue. i think this issue is about yupResolver.
<img width="971" alt="Screenshot 2022-01-03 at 2 56 16 PM" src="https://user-images.githubusercontent.com/50900838/147915920-08af18dc-9f27-446a-ac53-0e06d7cd54d0.png">

**Comment by bluebill1049** [comment, marked as answer]
update to the latest react-hook-form should resolve this issue

**Reply by vikas990** [reply]
but after updating the inputs are not registering?
